web_client/js/imageViewerWidget/base.js

```diff
@@ -8,6 +8,8 @@ girder.views.ImageViewerWidget = girder.View.extend({
         }).done(_.bind(function (resp) {
             this.levels = resp.levels;
             this.tileSize = resp.tileSize;
```

Comment:
Accordingly, go ahead and put a check in the initialization of Leaflet and have it fail if ``this.tileWidth != this.tileHeight``.